tests/CoreBundle/Form/Type/ColorTypeTest.php

```diff
@@ -32,9 +32,13 @@ public function testDeprecation()
             $this->markTestSkipped('< Symfony 3.4');
         }
 
-        $type = new ColorType();
+        $this->factory->create(ColorType::class);
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation The Sonata\CoreBundle\Form\Type\ColorType class is deprecated since version 3.10 and will be removed in 4.0. Use Symfony\Component\Form\Extension\Core\Type\ColorType instead.
```

Comment:
Do you know if there is a way to check this expectedDeprecation message without annotation (like the `TestCase::expectExceptionMessage` method)?
Actually this annotation makes the CI --prefer-lowest build fails.

Reply:
I think there isn't a way to achieve this. Note that the sf `ColorType` exists only recently in sf, which means that people using an old version of Symfony should not get a deprecation, which is probably what happens here… maybe you could use `$test->markAsSkipped()` instead?